Gemfile

```diff
@@ -70,6 +72,7 @@ gem "sanger_sequencing", "~> 0.0.1", path: "vendor/engines/sanger_sequencing"
 gem "secure_rooms", path: "vendor/engines/secure_rooms"
 gem "split_accounts", "~> 0.0.1", path: "vendor/engines/split_accounts"
 gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
+gem "net-telnet"
```

Comment:
Yeah, it caused an issue bundling unless Telnet was included, because of synaccess if I recall correctly. If we don't want to include this here, we could try updating synaccess, but I don't know what other projects use it/what the downstream consequences would be there.